src/RecordManager/Finna/Record/Ead3.php

```diff
@@ -323,7 +323,7 @@ protected function enrichTitlesWithYearRanges(
             if (!$yearRangeStr) {
                 return $data;
             }
-            $yearRangeStr = " ($yearRangeStr)";
+            $yearRangeStr = "\u{200E}" . " ($yearRangeStr)";
```

Comment:
Laita samaan stringiin, niin ei tarvitse turhaan yhdistää kahta. Voisi ehkä lisätä yläpuolelle myös kommentin tyyliin `// Append with LTR mark first to ensure correct text direction`